pybossa/exporter/task_csv_export.py

```diff
@@ -220,12 +220,12 @@ def _get_headers_from_row(self, obj, obj_name, expanded):
         headers = self.get_keys(obj_dict, obj_name)
         return headers
 
-    def _respond_csv(self, ty, project_id, expanded=False, filters=None):
+    def _respond_csv(self, ty, project_id, expanded=False, filters=None, disclose_gold=False):
```

Comment:
Are these changes also need to apply for json [here](https://github.com/bloomberg/pybossa/blob/1914-RDISCROWD/pybossa/exporter/task_json_export.py#L123)?

Reply:
Yes, I will make the changes there.